src/app/containers/PageHandlers/withData/shouldRender/index.js

```diff
@@ -0,0 +1,39 @@
+import pathOr from 'ramda/src/pathOr';
+import {
+  getPassportHome,
+  isValidPassportHome,
+} from '../../../../lib/utilities/passport';
+
+// checks for data, status, setting default status if not found
+const constructRenderObject = data => ({
+  status: pathOr(null, ['status'], data) || 500,
+  pageData: pathOr(null, ['pageData'], data),
+});
+
+// checks for pageData, 200 status and if home service from article data fits the service locale
+const shouldRender = (data, service, passportHomesOverride = []) => {
+  const { status, pageData } = constructRenderObject(data);
+
+  let statusCode = status;
+  let isCorrectService;
+
+  const hasDataAnd200Status = pageData && status === 200;
+  if (hasDataAnd200Status) {
+    const passportHome = getPassportHome(pageData);
+    isCorrectService = isValidPassportHome(passportHome, service);
```

Comment:
Why not just pass `passportHomesOverride` to `isValidPassportHome` and have it all deal with it inside that function?

Reply:
Done!